packages/core/src/dialects/db2/query-generator-typescript.ts

```diff
@@ -16,16 +16,21 @@ export class Db2QueryGeneratorTypeScript extends AbstractQueryGenerator {
     const table = this.extractTableDetails(tableName);
 
     return joinSQLFragments([
-      'SELECT NAME AS "Name", TBNAME AS "Table", TBCREATOR AS "Schema",',
-      'TRIM(COLTYPE) AS "Type", LENGTH AS "Length", SCALE AS "Scale",',
-      'NULLS AS "IsNull", DEFAULT AS "Default", COLNO AS "Colno",',
-      'IDENTITY AS "IsIdentity", KEYSEQ AS "KeySeq", REMARKS AS "Comment"',
-      'FROM',
-      'SYSIBM.SYSCOLUMNS',
-      `WHERE TBNAME = ${this.escape(table.tableName)}`,
-      'AND TBCREATOR =',
-      table.schema ? this.escape(table.schema) : 'USER',
-      ';',
+      'SELECT COLNAME AS "Name",',
+      'TABNAME AS "Table",',
+      'TABSCHEMA AS "Schema",',
+      'TYPENAME AS "Type",',
```

Comment:
I almost looked over this change. What I can find in https://www.ibm.com/docs/en/db2-for-zos/11?topic=tables-syscolumns is that it shouldn't change the behaviour unless `COLTYPE` was DISTINCT. But I can't find usages of that in DB2 so I think we should be fine

Reply:
This is actually a typo. Thanks for picking it up, I've changed this back to `TRIM(COLTYPE)` now.

Reply:
Ah, I didn't even notice that change. In that case it's good